formatTest/typeCheckedTests/expected_output/jsx.re

```diff
@@ -246,29 +246,29 @@ let spaceBefore2 = <So> <Much /> </So>;
 let siblingNotSpaced =
   <So> <Much /> <Much /> </So>;
 
-let jsxInList = [<Foo />];
+let jsxInList = [ <Foo /> ];
```

Comment:
It's kind of unfortunate to have to add spaces around everything like `[]` and `()` just because it's needed around `[| |]` in the *one* case where you use a self-closing tag.

Even `[|<X> </X>|]` parses correctly. It's just `[|<X />|]` that doesn't.